lib/tasks/data.rake

```diff
@@ -79,3 +79,32 @@ namespace :google_drive do
     puts "#{documents_deleted} Documents deleted. Have a good day!"
   end
 end
+
+namespace :vacancies do
+  desc "Fix about_school fields of live vacancies"
+  task fix_about_school: :environment do
+    organisation_ids = Organisation.not_closed.where.not(description: [nil, ""]).pluck(:id)
+
+    vacancies = Vacancy.live
+                       .distinct
+                       .joins(:organisation_vacancies)
+                       .where(organisation_vacancies: { organisation_id: organisation_ids })
+
+    fixed_vacancies = 0
+
+    vacancies.find_each do |vacancy|
+      organisation_description = vacancy.organisations.pluck(:description).compact.first
```

Comment:
Is it possible for `organisation_description` to be `nil`?

Reply:
added a check on that